examples/maa_lifecycle_example/main.go

```
@@ -0,0 +1,577 @@
+	binary.BigEndian.PutUint64(salt[3:11], uint64(time.Now().UnixNano()))
+	return salt, nil
+}
+
+// loadDotenv populates the environment from a KEY=VALUE .env file without overriding existing vars
+// (real environment variables take precedence). A missing file is a no-op.
+func loadDotenv(path string) {
+	f, err := os.Open(path)
+	if err != nil {
+		return
+	}
+	defer f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)